docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.7"
+
+services:
+  pgdb:
+    build: env/postgres
+    image: pgdb
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_PASSWORD=pw
```

Comment:
Sets the env var `POSTGRES_PASSWORD` that we had to set on the first day of setup. You'll need to change this to be whatever password you set.